ignek_intranet/themes/ignek-theme/src/templates/portal_normal.ftl

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+
+<#include init />
+
+<html class="${root_css_class}" dir="<@liferay.language key=" lang.dir" />" lang="${w3c_language_id}">
+
+<head>
+  <title>${html_title}</title>
+
+  <meta content="initial-scale=1.0, width=device-width" name="viewport" />
+
+  <@liferay_util["include"] page=top_head_include />
+</head>
+
+<body class="${css_class}">
+
+  <@liferay_ui["quick-access"] contentId="#main-content" />
+
+  <@liferay_util["include"] page=body_top_include />
+
+  <@liferay.control_menu />
+
+  <div class="container-fluid position-relative" id="wrapper">
+    <header id="banner" role="banner">
+      <div id="heading">
+        <div aria-level="1" class="site-title" role="heading">
+          <a class="${logo_css_class}" href="${site_default_url}" title="<@liferay.language_format arguments=" ${site_name}" key="go-to-x" />">
+          <img alt="${logo_description}" height="${site_logo_height}" src="${site_logo}" width="${site_logo_width}" />
+          </a>
+
+          <#if show_site_name>
+            <span class="site-name" title="<@liferay.language_format arguments=" ${site_name}" key="go-to-x" />">
+            ${site_name}
+            </span>
+          </#if>
+        </div>
+      </div>
+
+      <#if !is_signed_in>
+        <a data-redirect="${is_login_redirect_required?string}" href="${sign_in_url}" id="sign-in" rel="nofollow">${sign_in_text}</a>
+      </#if>
+
+      <div class="user-info">
+
+        <#assign userLocalService=serviceLocator.findService("com.liferay.portal.kernel.service.UserLocalService")>
```

Comment:
No need to call userLocalService. You can get the user object by themeDisplay.getUser(). And assign it in init_custom file.